packages/components/form/src/text_field.tsx

```diff
@@ -12,7 +12,7 @@ interface OwnProps {
   rules?: RegisterOptions;
 }
 
-export type TextFieldProps = OwnProps & Omit<InputProps, "theme">;
+export type TextFieldProps = OwnProps & Omit<InputProps, "theme"> & FlexProps;
```

Comment:
I believe in the RadioField update you mentioned we passed those props onto the wrapping component not the `input` itself. Right now we are splatting props on `Component` which will be either the `Input` or `TextArea` component.

We'll likely need to extract the sizing props and pass them onto the `FormField` component wrapping the `Component`.

Additionally, I'm not sure why there is a `Box` wrapping the `FormField` right now, I wonder if it can be removed 🤔